inference-engine/src/plugin_api/exec_graph_info.hpp

```diff
@@ -69,6 +69,12 @@ static const char EXECUTION_ORDER[] = "execOrder";
  */
 static const char LAYER_TYPE[] = "layerType";
 
+/**
+ * @ingroup ie_dev_exec_graph
+ * @brief Used to get runtime precisions of the executable primitive.
+ */
+static const char RUNTIME_PRECISION[] = "runtimePrecision";
```

Comment:
is it supposed we can have string of precision separated by `,`?

Reply:
No. We mean that runtime_precision is only one.
Do you think we need to change this line into `static const char RUNTIME_PRECISIONS[] = "runtimePrecisions";` ?

Reply:
@ilya-lavrenov, any updates on this?

Reply:
We need more input from @dmitry-gorokhov @vladimir-paramuzov @taka-no-me @mfedoro1

Reply:
@ilya-lavrenov there is a email thread on this and outcome was that single precision value is needed. If you now disagree then lets resume that email discussion.

Reply:
Ok, I agree. Just wanted to comfirm.

Reply:
Can we just explicitly state `Used to get a runtime precision of the executable primitive`? because currently it says about precisionS

Reply:
My bad. Fixed now.